cli/cli_cheats_sheet.tex

```diff
@@ -161,8 +164,7 @@
 \hrulefill
 
 %
-\clicmd{doctor} A tool to check ROS setup and other potential issues such as network, package versions,
-rmw middleware etc. \\
+\clicmd{doctor} Check ROS setup and other potential issues \\
```

Comment:
```suggestion
\clicmd{doctor} Check ROS setup and other potential issues. \\
```

Reply:
Fixed in https://github.com/ubuntu-robotics/ros2_cheats_sheet/pull/6/commits/2cb22ee86c7d261c14f6a32456d84f942ecac103